stl/inc/memory

```diff
@@ -3154,7 +3154,7 @@ protected:
         }
     }
 
-    _Ty* _Ptr = nullptr;
+    atomic<_Ty*> _Ptr{nullptr};

```

Comment:
Observation: This data member change is OK because it's C++20-only code. (No change requested)